script/brave_license_helper.py

```diff
@@ -168,4 +168,9 @@ def CheckBraveMissingLicense(target_os, path, error):
         else:
             if path in ANDROID_ONLY_PATHS:
                 return  # Android failures are not relevant on desktop.
+        print('\nERROR: missing license information under src/brave\n'
+              "If this is code you added, then you'll have to add the required "
+              "metadata.\nIf the path that's mentioned isn't something you "
+              "added, then you probably just need to remove that obsolete path "
+              "from your local checkout.\n")
```

Comment:
Would it be possible to print the path here? sometimes it can be tricky to find that with the error - that would be a nice thing to include along with the error message 😄

Reply:
I could do that. I hadn't included it because it's already in the existing error message:
```
__main__.LicenseError: missing README.chromium or licenses.py SPECIAL_CASES entry in brave/third_party/boost

[347/40646] CXX obj/base/base/scoped_thread_priority.o
```
but if you think it's worth printing in the top message too, I could add it.